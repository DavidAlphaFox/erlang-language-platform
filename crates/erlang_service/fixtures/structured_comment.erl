-module(structured_comment).

test(X) ->
    %$eqwalizer: dynamic(X)
    X.

test2(X) ->
    %$eqwalizer: X :: dynamic()
    X.
