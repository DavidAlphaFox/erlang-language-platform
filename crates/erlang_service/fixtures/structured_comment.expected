AST
{attribute,{0,0},file,{"fixtures/structured_comment.erl",0}}.
{attribute,{0,27},module,structured_comment}.
{function,{30,74},
          test,1,
          [{clause,{30,74},[{var,{35,36},'X'}],[],[{var,{73,74},'X'}]}]}.
{function,{77,126},
          test2,1,
          [{clause,{77,126},[{var,{83,84},'X'}],[],[{var,{125,126},'X'}]}]}.
{eof,{128,128}}.


WARNINGS
[
    ParseError {
        path: "fixtures/structured_comment.erl",
        location: Some(
            Normal(
                30..74,
            ),
        ),
        msg: "function test/1 is unused",
        code: "L1230",
    },
    ParseError {
        path: "fixtures/structured_comment.erl",
        location: Some(
            Normal(
                77..126,
            ),
        ),
        msg: "function test2/1 is unused",
        code: "L1230",
    },
]

ERRORS
[]
